interview-questions.md

```diff
@@ -8,44 +8,96 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Researched answer:
+Researched answer:A relational database is a database where data points can be related to other things. Kind of like if you have a sheet in excel, and it has a table that shows names of people and how much money they owe a company. If you click on a cell, the bar at the top will have a reference to another file with sheets and it had pulled the info from a totally different place than what you are looking at. If you change one of the sheets it can or will change the others depending on how you have it set up. Are there other types of databases, yes. I know that i have worked with some other ones but i am not sure of the names. I am sure there are types of databases that everyone has worked with. After further review it appears that i have worked with a personal database, because i made it. A personal database is a database that is usually stored on a personal computer and isnt really suitable for a complex organization. A relational database is the opposite of a NoSQL database. A great example that i read is that "a relational database would be the connection between a person shopping online and thier shopping cart." When i am on amazon, i am not physically putting things in my cart, i am just clicking it, it is not moving anything from the amazon warehouse list, it is just saying toss the info for this item into cliffs cart, and if he decides to purchase it, then we will go to the other part of our database and pull up cliffs credit card info and address and maybe his rewards points.
 
 5. What are primary keys? Why are they important?
 
-Your answer:
+Your answer: Primary keys used in databases as a unique identifier for a "xxx" in a model. It is like a social or a licence plate, it is used for specific "xxx" becuase other info in the model may be the same as others. in the example below i have created, the primary key for each of the "xxx"'s is different where as some of the info in diffferent coloumns is the same and different the primary keys are always different for this model. There will never be an overlap. 
+model:colors_of_life
+pk|first_name|last_name|favorite_color|eye_color|
+1|cliff|roecker|blue|blue
+2|john|roecker|red|brown
+3|mark|blast|blue|brown
 
 Researched answer:
+Primary keys used are used in relational databases as a unique identifier for a "xxx" in a model. It is like a social or a licence plate, it is used for specific "xxx" becuase other info in the model may be the same as others. in the example below i have created, the primary key for each of the "xxx"'s is different where as some of the info in diffferent coloumns is the same and different the primary keys are always different for this model. There will never be an overlap. 
+model:colors_of_life
+pk|first_name|last_name|favorite_color|eye_color|
+1|cliff|roecker|blue|blue
+2|john|roecker|red|brown
+3|mark|blast|blue|brown
```

Comment:
I would suggest explaining whether or you are using a certain syntax when writing the example. And when you use same and different explain the similarities and differences.